### PR TITLE
Remove app-secret values from demo apps

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Remove app-secret values from demo apps.

## Related PRs or issues

[AB#89925](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/89925)

### CODE REVIEW

**Review by DmitriyKirakosyan — APPROVED** [review]

File: AppCenterReactNativeShared/Products/local.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Pod::Spec.new do |s|
   s.name              = 'AppCenterReactNativeShared'
-  s.version           = '4.4.3'
+  s.version           = '4.4.1'
```
Comment (DmitriyKirakosyan) [inline AppCenterReactNativeShared/Products/local.podspec:6]: Why? I believe it should be 4.4.3.
as everywhere:

https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/package.json#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-crashes/package.json#L3
https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter/package.json#L3

Comment (AnastasiaKubova) [reply AppCenterReactNativeShared/Products/local.podspec:6]: @AnatolyPristensky could you please explain?

Comment (AnatolyPristensky) [reply AppCenterReactNativeShared/Products/local.podspec:6]: @DmitriyKirakosyan leaving it on 4.4.3 causing the following error on CI: 
```
CocoaPods could not find compatible versions for pod "AppCenterReactNativeShared":
  In snapshot (Podfile.lock):
    AppCenterReactNativeShared (from `../../AppCenterReactNativeShared/Products/local.podspec`)
  In Podfile:
    AppCenterReactNativeShared (from `../../AppCenterReactNativeShared/Products/local.podspec`)
```

I'm not sure why, but looks like that local.podspec should contain version of Apple SDK.

Comment (DmitriyKirakosyan) [reply AppCenterReactNativeShared/Products/local.podspec:6]: @AnatolyPristensky  local.podspec should not be used on CI. It is for local build only. Please check why it gets used on CI.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**